opcode/opcode.go

```diff
@@ -109,7 +109,7 @@ var opsLiteral = map[Op]string{
 	RightShift: ">>",
 	GE:         ">=",
 	LE:         "<=",
-	EQ:         "==",
+	EQ:         "=",
```

Comment:
@tiancaiamao  PTAL
Will there be a problem with this modification?

Reply:
It fix a bug, I think.

Reply:
@leoppro Maybe I'd better create another pull request to fix this first?